servers/cromwell/jobs/controllers/jobs_controller.py

```diff
@@ -128,29 +129,88 @@ def get_job(id, **kwargs):
 
 
 def format_task(task_name, task_metadata):
+    # check to see if task is scattered
+    is_scattered_task = len(
+        set([item.get('shardIndex') for item in task_metadata])) > 1
+    if is_scattered_task:
+        return format_scattered_task(task_name, task_metadata)
+    else:
+        latest_attempt = task_metadata[-1]
+        return TaskMetadata(
+            name=remove_workflow_name(task_name),
+            execution_id=latest_attempt.get('jobId'),
+            execution_status=task_statuses.cromwell_execution_to_api(
+                latest_attempt.get('executionStatus')),
+            start=_parse_datetime(latest_attempt.get('start')),
+            end=_parse_datetime(latest_attempt.get('end')),
+            stderr=latest_attempt.get('stderr'),
+            stdout=latest_attempt.get('stdout'),
+            inputs=update_key_names(latest_attempt.get('inputs', {})),
+            return_code=latest_attempt.get('returnCode'),
+            attempts=latest_attempt.get('attempt'),
+            call_root=latest_attempt.get('callRoot'),
+            job_id=latest_attempt.get('subWorkflowId'),
+            shard_statuses=None)
+
+
+def format_scattered_task(task_name, task_metadata):
+    temp_status_collection = {}
+    current_shard = ''
+    minStart = _parse_datetime(task_metadata[0].get('start'))
+    maxEnd = _parse_datetime(task_metadata[0].get('end'))
+
+    # go through calls in reverse to grab the latest attempt if there are multiple
+    for shard in task_metadata[::-1]:
+        if current_shard != shard.get('shardIndex'):
+            status = task_statuses.cromwell_execution_to_api(
+                shard.get('executionStatus'))
+            if status not in temp_status_collection:
+                temp_status_collection[status] = 1
+            else:
+                temp_status_collection[
+                    status] = temp_status_collection[status] + 1
+        current_shard = shard.get('shardIndex')
+        if minStart > _parse_datetime(shard.get('start')):
+            minStart = _parse_datetime(shard.get('start'))
+        if maxEnd < _parse_datetime(shard.get('end')):
+            maxEnd = _parse_datetime(shard.get('end'))
```

Comment:
If one shard is done and one is still running, does it make sense to display the end-time?